docs/getting_started.rst

```diff
@@ -34,3 +34,110 @@ If it has been a while, once can update using
 
     conda activate garnet
     conda env update --file environment.yml --prune
+
+
+Testing
+```````````````````
+The project contains testing infrastructure in the tests/ folder:
+- data/ : It contains the test data. Small data files can be included in this directory. Forl large data files, there is a subfolder garner-data
+ that is connected to a gitlfs data repo in gitlab as a submodule: https://code.ornl.gov/sns-hfir-scse/infrastructure/test-data/garnet-data
+- tabs/ : Tests are organized into folders for every page:
+    * home/
+    * order_param/
+    * peak_integr/
+    * recp_space/
+    * ub_peak_find/
+
+        In every page-tab directory the tests are furtherd arranged in MVP-based filesystem structure:
```

Comment:
![Screenshot from 2024-02-20 21-01-59](https://github.com/neutrons/garnet/assets/118226485/b4ea4259-c18d-4b7d-a0a7-1b5912ddca52) Is this how you intended it to be displayed? Sphinx may read this file different but in Github this sort of looks unintended.

Reply:
nope. done